consensus/src/member.rs

```diff
@@ -135,6 +136,66 @@ impl<D: Data, DP: DataProvider<D>, FH: FinalizationHandler<D>, US: Write, UL: Re
     }
 }
 
+struct MemberStatus<'a, H, D, S>
```

Comment:
Eh, I saw the other instance and perhaps for consistency this can stay as a struct, but then please make a constructor rather than `pub`ing all fields.

Reply:
removed `pub`s